pycode/memilio-epidata/memilio/epidata_test/test_epidata_getHospitalizationData.py

```diff
@@ -146,10 +146,8 @@ def test_compute_hospitailzations_per_day(self):
                 except:
                     pass
         # should be equal to the input array
-        self.assertAlmostEqual(
-            list(inputarray_r[6:]),
-            list(control_sum[6:]),
-            10)
+        np.testing.assert_array_almost_equal(
```

Comment:
Where is correlation with issue #554 ?

Reply:
There is no direct correlation. The epidata CI failed because `assertAlmostEqual` gets confused with lists/arrays if they're not exactly equal

Reply:
so in the future this should be addresed in a seperate issue :)